src/providers/RemoteVideoTileProvider/state.tsx

```diff
@@ -1,18 +1,12 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-type tileMap = {
-  [key: string]: string;
```

Comment:
Since this is tileId -> attendeeId so  `[key: number]: string;`? Will save those `toString()`s.